app/views/candidate_interface/gcse/type/_form.html.erb

```diff
@@ -5,7 +5,7 @@
   hint: { text: @subject == 'english' ? 'This should not be a qualification showing you speak English as a foreign language.' : '' } do %>
 
   <% select_gcse_qualification_type_options.each_with_index do |option, i| %>
-    <%= f.govuk_radio_divider if i == select_gcse_qualification_type_options.count - 1 %>
+    <%= f.govuk_radio_divider if i >= select_gcse_qualification_type_options.count - 2 %>
```

Comment:
This feels kinda icky? Any ideas how it could be improved?

Reply:
Best bet I think would be to change `select_gcse_qualification_type_options` to not include the "special" ones, and then add those outside of the loop with their own dividers.

Reply:
@elliotcm makes sense. I had a quick go at this but it was a bigger change than I thought and so I’ve backed out. 😄  I'll ask you or a dev to help next week!

Reply:
@carlosmartinez 🙏